packages/typescript/src/lib/TypescriptCompiler.ts

```diff
@@ -130,6 +130,9 @@ export class TypescriptCompiler {
         const importPath =
           './' + relativePathToCompiledFile.replace(/\.d\.ts$/, '');
         const reexport = `export * from '${importPath}';\nexport { default } from '${importPath}';`;
+
+        this.tsDefinitionFilesObj[`${this.options.distDir}/${exposedDestFilePath.replace('./', '')}.d.ts`] = reexport;
```

Comment:
```suggestion
        this.tsDefinitionFilesObj[normalizedExposedDestFilePath] = reexport;
```

Reply:
Hey there, `normalizedExposedDestFilePath` isn't formatted the same as the `filepath` arg passed to `host.writeFile`,

`/Users/me/root-dir/dist/@mf-types/Exposed.d.ts` vs `dist/@mf-types/_types/sub-dir/index.d.ts`,

so the consuming remote is throwing a 404 when trying to find the file.

Any thoughts?